src/Order.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+function Order({ orderType }) {
+  const [orders, setOrders] = useState(0);
+  function orderOne() {
+    setOrders(orders + 1);
+  }
+  return (
+    <li className="order-list">
+      {orderType}: {orders} <RestaurantButton eventHandler={orderOne} />
```

Comment:
I'd avoid calling it `eventHandler` as that is a very general name for a function  - indeed it is true of any function that gets called when an event occurs. Instead I'd think about using a more descriptive name relevant to your app and what it does to the `orders`.